soccer/src/soccer/coach/coach_node.cpp

```diff
@@ -0,0 +1,127 @@
+#include "coach_node.hpp"
+
+CoachNode::CoachNode(const rclcpp::NodeOptions& options) : Node("coach_node", options) {
+    coach_state_pub_ =
+        this->create_publisher<rj_msgs::msg::CoachStateInterpretation>("/strategy/coach_state", 10);
+    play_state_change_timer_ =
+        this->create_wall_timer(100ms, [this]() { check_for_play_state_change(); });
+
+    play_state_sub_ = this->create_subscription<rj_msgs::msg::PlayState>(
+        "/referee/play_state", 10,
+        [this](const rj_msgs::msg::PlayState::SharedPtr msg) { play_state_callback(msg); });
```

Comment:
```suggestion
        [this](const rj_msgs::msg::PlayState::SharedPtr &msg) { play_state_callback(msg); });
```
90% sure this will fix this check warning. don't forget to change the .hpp to match

Reply:
same comment for the other warnings of similar nature